backend/src/models/mentor.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * This file contains the model for the mentors. Note
+ * personal information such as email and password will
+ * be stored on firebase
+ */
+import mongoose from "mongoose";
+
+interface MentorInterface {
+  name: string;
+  organization_id: string;
+  access_token: string;
+  status: string;
+}
+
+interface MentorDoc extends mongoose.Document {
+  name: string;
+  organization_id: string;
+  access_token: string;
+  status: string;
+}
+
+interface MentorModelInterface extends mongoose.Model<MentorDoc> {
+  build(attr: MentorInterface): MentorDoc;
+}
+
+const mentorSchema = new mongoose.Schema({
```

Comment:
After looking at Figma I realize that we need to update mentor and mentee schema to store additional values (images, major, interests, etc.). Not important now but it will be updated in future PR's.

Reply:
It is kinda important so that the the frontend can be properly integrated with the Signup flow. Since this shouldn't take too long, can we add the fields?

Reply:
to be resolved - https://github.com/TritonSE/ALUM-Mobile-Application/issues/46